packages.dot

```diff
@@ -1,6 +1,7 @@
 digraph packages {
 	"/service/cluster" -> "/service/k8sclient";
 	"/service/cluster" -> "/service/k8sclient/common";
+	"/service/k8sclient" -> "";
```

Comment:
what does it mean?

Reply:
https://github.com/percona-platform/dbaas-controller/pull/184/files/2f204d13c732beb716338437814b45542d429604#diff-68d507467191c86afbfbd5f639a2f013e2a5a15a8050bf2af591f1cbcce1a19eR36
I think that line is the reason. It is not intuitive but makes sense.